### DISCUSSION TITLE
Getting Local Versions of n8n and Supabase to work together

### DISCUSSION THREAD

**Discussion by PattrnData** [supabase/supabase#30982] [Questions]
Hey everyone i have setup n8n and Supabase on my local server with all the relevant API keys, secrets and tokens. Using curl i can use the service role token.

However when in n8n when adding the credentials no matter using the service role or anon key it gives me an error of unauthorised

<img width="696" alt="image" src="https://github.com/user-attachments/assets/a6abec4f-b3ed-4e78-abed-759089bc42d1">

Does anyone have any idea of how to fix this?

**Comment by rsshilli** [comment]
I got it working with this: https://www.reddit.com/r/Supabase/comments/18b4znv/unable_to_connect_supabase_with_n8n_on_docker/
